internal/handler/handler.go

```diff
@@ -192,7 +192,6 @@ func (h *Handler) cleanupAllCappedCollections(ctx context.Context) error {
 	}()
 
 	connInfo := conninfo.New()
-	connInfo.BypassBackendAuth = true
```

Comment:
Does cleaning capped collection work without this?

Reply:
That was mistakenly removed and not readded. I added it back.